src/Juno/JunoClient.ts

```diff
@@ -208,6 +208,24 @@ export class JunoClient {
     };
   }
 
+  public async isCodeOfConductAccepted(): Promise<IJunoResponse<boolean>> {
+    const input = `${this.getNotebooksAccountUrl()}/gallery/isCodeOfConductAccepted`;
+    const response = await window.fetch(input, {
+      method: "PATCH",
+      headers: JunoClient.getHeaders()
+    });
+
+    let data: boolean;
+    if (response.status === HttpStatusCodes.OK) {

```

Comment:
You can check `response.ok` here

Reply:
this is the general pattern followed here. Will make these changes as part of a clean up PR.